changes for building with gnome-desktop 44.0

drop when updating gnome-control-center to a newer version

--- a/panels/background/cc-background-chooser.c	2024-01-20 08:57:08.645685229 -0800
+++ b/panels/background/cc-background-chooser.c	2024-01-20 08:57:29.640833859 -0800
@@ -265,7 +265,7 @@
         {
           pixbuf = gnome_desktop_thumbnail_factory_generate_thumbnail (thumbnail_factory,
                                                                        uri,
-                                                                       mime_type);
+                                                                       mime_type, NULL, NULL);
         }
 
       gtk_dialog_set_response_sensitive (GTK_DIALOG (chooser),
